examples/datasource-streaming-websocket/streaming-websocket-plugin/src/DataSource.ts

```diff
@@ -65,9 +67,6 @@ export class DataSource extends DataSourceApi<MyQuery, MyDataSourceOptions> {
   }
 
   filterQuery(query: MyQuery): boolean {
-    if (query.hide) {
-      return false;
-    }
-    return true;
+    return !!query.queryText;
```

Comment:
I think queryText is not used in this plugin

Reply:
Maybe it shouldn't be, but it is [defined](https://github.com/grafana/grafana-plugin-examples/blob/375cc7ec9eae4f7d712172b13a361ef6818d0377/examples/datasource-streaming-websocket/streaming-websocket-plugin/src/types.ts#L4) in types.